core/src/main/java/com/linecorp/armeria/client/retry/RetryingHttpClientBuilder.java

```diff
@@ -59,16 +73,21 @@ public RetryingHttpClientBuilder useRetryAfter(boolean useRetryAfter) {
     }
 
     /**
-     * Sets the length of content to look up whether retry or not. If the total length of content exceeds
+     * Sets the length of content to look up whether retry or not when you create this builder with a
+     * {@link RetryStrategyWithContent}. If the total length of content exceeds
      * this and there's no retry condition matched, it will hand over the stream to the client.
      * @param contentPreviewLength the content length to preview. Unlike other length related arguments,
-     *                             {@code 0} does not disable the length limit. It means that it will not
-     *                             look up the content
+     *                             {@code 0} does not disable the length limit.
+     *
      * @return {@link RetryingHttpClientBuilder} to support method chaining
+     *
+     * @throws IllegalStateException if this builder is created with a {@link RetryStrategy}
+     * @throws IllegalArgumentException if the specified {@code contentPreviewLength} is 0
```

Comment:
- 0 is a literal and thus has to be `{@code 0}`.
- `is` -> `is equal to or less than`